tests/integration-tests/tests/tags/test_tag_propagation.py

```diff
@@ -40,17 +48,45 @@ def test_tag_propagation(pcluster_config_reader, clusters_factory, scheduler, os
     - compute node's root EBS volume (traditional schedulers)
     - shared EBS volume
     """
-    config_file_tags = {"ConfigFileTag": "ConfigFileTagValue"}
-    version_tags = {"parallelcluster:version": get_pcluster_version()}
     cluster_config = pcluster_config_reader()
     cluster = clusters_factory(cluster_config)
+
+    # Uses helper function to perform the tests
+    _tag_propagation_checker_helper(cluster, scheduler, os)
+
+    if scheduler == "slurm":
+        cluster.stop()
+
+    # Updates cluster with new configuration
+    updated_cluster_config = pcluster_config_reader(config_file="pcluster.config.update.yaml")
+    cluster.update(str(updated_cluster_config), force_update="true")
+
+    if scheduler == "slurm":
+        cluster.start()
+
+    # Makes sure that the compute nodes have started before checking tags
+    _wait_for_compute_fleet_start(cluster)
+
+    # Checks for tag propagation
+    _tag_propagation_checker_helper(cluster, scheduler, os)
+
+
+@retry(wait_fixed=seconds(20), stop_max_delay=minutes(8))
+def _wait_for_compute_fleet_start(cluster):
+    compute_nodes = cluster.get_cluster_instance_ids(node_type="Compute")
+    assert_that(len(compute_nodes) >= 1).is_true()
```

Comment:
Shall we assert the number of compute_nodes is equal to the min_count here?

Reply:
Done!